Changes

```diff
@@ -54,6 +54,10 @@ Working version
 - #9582: Add Array.{find_opt,find_map,split,combine}.
   (Nicolás Ojeda Bär, review by Daniel Bünzli and Gabriel Scherer)
 
+- #10192: Add support for Unix domain sockets on Windows and use them
+  to emulate Unix.socketpair (only available on Windows 1803+)
+  (Antonin Décimo, review by David Allsopp)
+
```

Comment:
This either be in `Runtime system` or `Other libraries` but definitely not `Standard library`. I'd go with `Other libraries` - the runtime change is for the debugger's benefit; the support in Unix is the primary interesting part!

Reply:
Moved to _Other libraries_!